src/Mutex/PostgreSQLMutex.php

```diff
@@ -13,9 +14,12 @@ class PostgreSQLMutex extends AbstractLockMutex
     /** @var array{int, int} */
     private array $key;
 
-    public function __construct(\PDO $PDO, string $name)
+    private ?float $timeout;
+
+    public function __construct(\PDO $PDO, string $name, ?float $timeout = null)
```

Comment:
mimic the naming/defaults convenction https://github.com/php-lock/lock/blob/3.0.2/src/Mutex/FlockMutex.php#L36